src/met-atlas-viewer.js

```diff
@@ -365,6 +365,14 @@ const MetAtlasViewer = targetElement => {
     });
   };
 
+  const updateNodeColors = async colorData => {
```

Comment:
Is `async` needed here?

Reply:
Hm, no, I can't see any reason it would be needed, and it seems to be working without it. Removed in 8e0a236.